experimental/css.html

```diff
@@ -0,0 +1,127 @@
+<style>
+
+  .branding-below {
+    bottom: 56px;
+    top: 0;
+  }
+  
+  .heading {
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 0;
+    margin-inline-end: 0;
+  }
+  
+  .btn-add {        
+    display: inline;
+    float: right; 
+    height: 20px; 
+    line-height: 20px;
+    min-width: 40px; 
+    max-width: 40px; 
+  }
+  
+  .btn-delete1 {     
```

Comment:
Add a description to the css class. how is it different from btn-delete2.

Reply:
In the new spreadsheet I have removed this classes and added classes for icons with appropriate names.